mrcnn/model.py

```diff
@@ -2171,14 +2550,14 @@ def compile(self, learning_rate, momentum):
             if layer.output in self.keras_model.losses:
                 continue
             loss = (
-                tf.reduce_mean(layer.output, keepdims=True)
+                tf.reduce_mean(layer.output, keep_dims=True)
```

Comment:
This change is unnecessary. `keep_dims` is deprecated.

Reply:
Cool I'll remove this from the commit